src/lib/flow/include/sol-flow-parser.h

```diff
@@ -72,11 +73,10 @@ struct sol_flow_parser_client {
      * @param data Client's data
      * @param name File name to read
      * @param buf Where to store the file
```

Comment:
Explicitly say that the buffer must be already initialized and data will be appended. If the data does not fit, an error is returned